invenio_pages/services/permissions.py

```diff
@@ -15,8 +15,8 @@
 class PagesPermissionPolicy(BasePermissionPolicy):
     """Permission policy for pages."""
 
-    can_create = []
+    can_create = [SystemProcess()]
```

Comment:
Minor: why this change? In reallity both (creation and deletion) are disabled, or am I missing something?

Reply:
Because the create and delete services need to be called by the fixtures of invenio-app-rdm